projects/plugins/jetpack/modules/google-fonts/current/class-jetpack-google-font-face.php

```diff
@@ -122,7 +122,9 @@ public function collect_global_styles_fonts() {
 	public function collect_block_fonts( $content, $parsed_block ) {
 		if ( ! is_admin() && isset( $parsed_block['attrs']['fontFamily'] ) ) {
 			$block_font_family = $parsed_block['attrs']['fontFamily'];
-			$this->add_font( $block_font_family );
+			if ( is_string( $block_font_family ) ) {
+				$this->add_font( $block_font_family );
```

Comment:
How about checking the value inside the `add_font` function? I think it's safer

Reply:
Done!